src/Covid19Dashboard/index.jsx

```diff
@@ -417,7 +419,7 @@ Covid19Dashboard.propTypes = {
 Covid19Dashboard.defaultProps = {
   modeledFipsList: [],
   jhuGeojsonLatest: { type: 'FeatureCollection', features: [] },
-  jhuJsonByLevelLatest: { country: {}, state: {}, county: {}, last_updated: '' },
+  jhuJsonByLevelLatest: { country: {}, state: {}, county: {} },
```

Comment:
```suggestion
  jhuJsonByLevelLatest: { country: {}, state: {}, county: {}, last_updated: '' },
```
I believe this was an over site in my merge

Reply:
put back in